tests/websockets_SUITE.erl

```diff
@@ -67,7 +67,9 @@ chat_msg(Config) ->
     escalus:story(Config, [{alice, 1}, {geralt, 1}, {oldie, 1}], fun(Alice, Geralt, Oldie) ->
 
         escalus_client:send(Alice, escalus_stanza:chat_to(Geralt, <<"Hi!">>)),
-        escalus:assert(is_chat_message, [<<"Hi!">>], escalus_client:wait_for_stanza(Geralt)),
+        FromAlice = escalus_client:wait_for_stanza(Geralt),
+        escalus:assert(is_chat_message, [<<"Hi!">>], FromAlice),
+        escalus:assert(has_ns, [<<"jabber:client">>], FromAlice),
```

Comment:
There is no need because oldie uses the "old" WS spec where the stream was started as in TCP connection so the xmlns can be omitted (at least in my opinion). Moreover the support for the old spec should be dropped in future.